internal/generator/vector/output/syslog/syslog.go

```diff
@@ -170,23 +219,53 @@ func getEncodingTemplatesAndFields(s *obs.Syslog) EncodingTemplateField {
 	templateFields := EncodingTemplateField{
 		FieldVRLList: []FieldVRLStringPair{},
 	}
-	if s.AppName != "" {
+
+	if s.Facility == "" {
 		templateFields.FieldVRLList = append(templateFields.FieldVRLList, FieldVRLStringPair{
-			Field:     "app_name",
-			VRLString: commontemplate.TransformUserTemplateToVRL(s.AppName),
+			Field:     "facility",
+			VRLString: commontemplate.TransformUserTemplateToVRL(`{._internal.syslog.facility || "user"}`),
 		})
 	}
-	if s.MsgId != "" {
+
+	if s.Severity == "" {
 		templateFields.FieldVRLList = append(templateFields.FieldVRLList, FieldVRLStringPair{
-			Field:     "msg_id",
-			VRLString: commontemplate.TransformUserTemplateToVRL(s.MsgId),
+			Field:     "severity",
+			VRLString: commontemplate.TransformUserTemplateToVRL(`{._internal.syslog.severity || "informational"}`),
 		})
 	}
 
-	if s.ProcId != "" {
+	if s.ProcId == "" {
+		s.ProcId = `{._internal.syslog.proc_id || "-"}`
```

Comment:
My previous comment regarding "default" applies and the reason I really want to hash this out in the document of what the various scenarios look like.  IMO the fallback should not be `-`; it should be empty

Reply:
yes for RFC3164 should be empty, but for RFC5164 `-`. I will update